android-test/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
     versionName "1.0"
 
     testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    testInstrumentationRunnerArguments(['notClass': 'org.conscrypt.KitKatPlatformOpenSSLSocketImplAdapter'])
+    testInstrumentationRunnerArguments(['notClass': 'org.conscrypt.KitKatPlatformOpenSSLSocketImplAdapter,org.bouncycastle.pqc.crypto.qtesla.QTeslaKeyEncodingTests'])
```

Comment:
such a weird thing to have to defend against

Reply:
Yep.

It's in their bcprov Jar.  We depend on that strongly via okhttp-testing-support and then it get's picked up by the instrumentation runner.